### PR TITLE
feat: wishlist page

### PR DESCRIPTION
**Pull request by PuneetP16**
## Wishlist Page
image

### CODE REVIEW

**Comment by sushmita12345** [comment]
Clean code. Nice UI, looks good to merge.

**Review by sushmita12345 — APPROVED** [review]

**Review by m26unkwn — APPROVED** [review]
Good to merge.
everything is good to me.

File: pages/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.main--wishlist .card .like {
+	background-color: var(--COLOR-PRIMARY);
+	color: var(--COLOR-WHITE);
+}
+
+/* To change on click in future */
+/* .main--wishlist .card .like:hover {
+	border: 2px solid var(--COLOR-PRIMARY);
```
Comment (rahulyadav139) [inline pages/wishlist/wishlist.css:53]: Remove unnecessary comments.

File: pages/wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+.main--wishlist {
+	display: flex;
+	flex-flow: column nowrap;
+	padding: 2rem 10rem;
+	gap: 3rem;
+}
+
+/*-------------------- main wishlist ------------------------ */
+
+.main--wishlist .section__heading {
```
Comment (rahulyadav139) [inline pages/wishlist/wishlist.css:36]: Class names are simple and descriptive. Great work. Go ahead and merge the branch.

**Review by rahulyadav139 — APPROVED** [review]

**Review by dhruvin25799 — APPROVED** [review]
Clean UI, very clean code, love the class names, short descriptive and apt.
Good to merge

File: pages/wishlist/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+/* .main--wishlist .card .like:hover {
+	border: 2px solid var(--COLOR-PRIMARY);
+    color: var(--COLOR-PRIMARY);
+	background-color: var(--COLOR-WHITE);
+} */
+
+/* Load if wishlist empty */
+.wishlist_empty {
+	display: flex;
+	flex-flow: column nowrap;
```
Comment (dev-ashishsethi) [inline pages/wishlist/wishlist.css:61]: good to see that you are using flex flow

File: pages/wishlist/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+.body--wishlist {
+	display: grid;
+	min-height: 100vh;
+	grid-template-columns: 1fr;
+	grid-template-rows: 7rem 1fr;
+	grid-template-areas:
+		"header"
+		"main";
+	transition: all 0.3s ease-in;
```
Comment (dev-ashishsethi) [inline pages/wishlist/wishlist.css:22]: 0.2 seconds would be better

**Review by dev-ashishsethi — APPROVED** [review]
nice work

File: pages/wishlist/wishlist.css (hunk 4 above)
Comment (PuneetP16) [reply pages/wishlist/wishlist.css:22]: I will try that. Thanks for reviewing.

File: pages/wishlist/wishlist.css (hunk 3 above)
Comment (PuneetP16) [reply pages/wishlist/wishlist.css:61]: Yeah! I try to use shorthand properties as much as possible.

File: pages/wishlist/wishlist.css (hunk 2 above)
Comment (PuneetP16) [reply pages/wishlist/wishlist.css:36]: Thank you for mentioning.

File: pages/wishlist/wishlist.css (hunk 1 above)
Comment (PuneetP16) [reply pages/wishlist/wishlist.css:53]: It have added this for my use case.

**Pull request merged**